incubator/couchdb/templates/statefulset.yaml

```diff
@@ -72,8 +72,8 @@ spec:
 {{ toYaml .Values.resources | indent 12 }}
           volumeMounts:
           - name: config-storage
-            mountPath: /opt/couchdb/etc/default.d
-            subMountPath: chart.ini
+            mountPath: /opt/couchdb/etc/default.d/chart.ini
+            subPath: chart.ini
```

Comment:
Any reason why we need a `subPath` at all? I think it should be like this without `subPath`:

```yaml
mountPath: /opt/couchdb/etc/default.d
```

Reply:
it's needed if there are other files in the /opt/couchdb/etc/default.d directory. If there are files that would be there then using the mountPath and pulling in an emptyDir would wipe them out. Using the file name in the mountPath and subPath guards against that. If you feel it is safe then I can remove the file name from the mountPath. @unguiculus

Reply:
Thanks for the clarification @mainephd. I'm comfortable with this restriction; we can make sure that the config settings in `default.d` that are specified in the upstream image are included as defaults in the `values.yaml`.